### ISSUE TITLE
Switch to the new frame

### ISSUE THREAD

**Issue by elzbietafuchs** [atata-framework/atata#11]
Hello,
How can i switch to the new iframe on Atata? 
Ela

**Comment by YevgeniyShunevych** [comment]
Hi,

Basically, you can use WebDriver functionality: `Driver.SwitchTo().Frame(...)`. Here are few examples:

### Control as iframe
An example of rich text editor control that works as `iframe` (TineMCE, etc.):
```cs
using Atata;
using OpenQA.Selenium;

namespace SampleApp
{
    [ControlDefinition("iframe")]
    public class RichTextBox<_> : EditableField<string, _>
        where _ : PageObject<_>
    {
        protected override string GetValue()
        {
            var frame = Driver.SwitchTo().Frame(Scope);
            string valueAsString = frame.Get(By.TagName("body")).Text;

            Driver.SwitchTo().DefaultContent();
            return ConvertStringToValue(valueAsString);
        }

        protected override void SetValue(string value)
        {
            string valueAsString = ConvertValueToString(value);

            var frame = Driver.SwitchTo().Frame(Scope);
            frame.Get(By.TagName("body")).SendKeys(valueAsString);

            Driver.SwitchTo().DefaultContent();
        }
    }
}
```

### Page object as iframe
An example of page object for page that is located inside the other one as `iframe`:
```cs
using Atata;
using OpenQA.Selenium;
using _ = SampleApp.SomePage;

namespace SampleApp
{
    public class SomePage : Page<_>
    {
        protected override void OnInit()
        {
            base.OnInit();

            IWebElement iframe = Driver.Get(By.XPath(".//iframe[@id='some-id']"));
            Driver.SwitchTo().Frame(iframe);
        }

        protected override void CleanUp()
        {
            base.CleanUp();

            Driver.SwitchTo().DefaultContent();
        }
    }
}
```

**Comment by YevgeniyShunevych** [comment]
I have released [Atata 0.12.0](https://www.nuget.org/packages/Atata/). Added few approaches for iframe interaction which you may check:
* [Frame control](https://github.com/atata-framework/atata/issues/21)
* [SwitchToFrame and SwitchToRoot methods of PageObject`1](https://github.com/atata-framework/atata/issues/20)
